agent.go

```diff
@@ -405,12 +402,22 @@ func (a *Agent) connectivityChecks() {
 	}
 
 	for {
+		interval := a.taskLoopInterval
+		switch lastConnectionState {
+		case ConnectionStateNew, ConnectionStateChecking: // While connecting, check candidates more frequently
+			interval /= 10
```

Comment:
I think it's better to add a new config parameter to specify ping interval during checking.
Then, interval can be choosed like
```go
var interval time.Duration
switch lastConnectionState {
case ConnectionStateNew, ConnectionStateChecking: // While connecting, check candidates more frequently
	interval = a.checkInterval
case ConnectionStateConnected, ConnectionStateDisconnected:
	interval = a.keepaliveInterval
default:
	// ConnectionStateFailed
	return
}
```